### PR TITLE
First Negative Integer In Every Window Of Size K in JavaScript

### PR DESCRIPTION
**Pull request by Coder-Srinivas**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Given an array of integers and a an integer k, we are supposed to return an array 
which contains the first negative integer in every window of size k 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue: #2304

### CODE REVIEW

**Comment by Coder-Srinivas** [comment]
@siddharth25pandey @HarshCasper I have added the code for First Negative Integer In Every Window Of Size K in JavaScript. You can test it [here](https://practice.geeksforgeeks.org/problems/first-negative-integer-in-every-window-of-size-k/0#). Please let me know if any changes are required. 😊

File: JavaScript/cp/FirstNegativeIntegerInEveryWindowOfSizeK.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            ans[i - k + 1] = 0;
+        }
+
+    }
+
+    return ans;
+}
+
+/* Printing the first negative integer in every window of size k */
+console.log(firstNegativeIntegerInEveryWindowOfSizeK([12, -1, -7, 8, -15, 30, 16, 28], 3))
```
Comment (HarshCasper) [inline JavaScript/cp/FirstNegativeIntegerInEveryWindowOfSizeK.js]: Make it as a dynamic input

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this PR due to inactivity.

**Comment by Coder-Srinivas** [comment]
@HarshCasper I have added dynamic input using the readline() module. Please let me know if any changes are required.

File: JavaScript/cp/FirstNegativeIntegerInEveryWindowOfSizeK.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+Example 2:
+
+Input:
+
+nums = [1, 2, 3, 4, 5]
+k = 5
+
+Output:
+[0]
```
Comment (ankitaggarwal23) [inline JavaScript/cp/FirstNegativeIntegerInEveryWindowOfSizeK.js]: Write a proper display message how you are reading nums , k and printing output.
Remove the extra testcase one is enough.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by Coder-Srinivas** [comment]
@ankitaggarwal23 Made the changes requested.

File: JavaScript/cp/FirstNegativeIntegerInEveryWindowOfSizeK.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+Example 1:
+
+Input:
+
+nums = [-8, 2, 3, -6, 10]
+k = 2
+
+Output:
+
+[-8, 0, -6, -6]
```
Comment (ankitaggarwal23) [inline JavaScript/cp/FirstNegativeIntegerInEveryWindowOfSizeK.js]: Write the I/p and O/p the way your code is producing

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by Coder-Srinivas** [comment]
@ankitaggarwal23 Made the changes requested.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**